src/sentry/grouping/enhancer/matchers.py

```diff
@@ -268,6 +268,13 @@ class CategoryMatch(FrameFieldMatch):
 
 
 class ExceptionFieldMatch(FrameMatch):
+    def matches_frame(self, frames, idx, platform, exception_data, cache):
+        match_frame = None
+        rv = self._positive_frame_match(match_frame, platform, exception_data, cache)
+        if self.negated:
+            rv = not rv
+        return rv
```

Comment:
This override ensures that exception matchers now ignore the frame index passed through `idx` (see `FrameMatch.matches_frame` for the original implementation).

Ideally though, `ExceptionFieldMatch` would not even derive from `FrameMatch`.